### PR TITLE
Various consistency and simplicity changes

### PR DESCRIPTION
**Pull request by nklayman**
I went through the getting started and usage (excluding Patterns) sections, and made a bunch of various changes that improve consistency and simplify a lot of wording. I know a lot of my changes are subjective, but I think I'm very close to the average person coming to Tauri. I have minimal Rust experience, but lots of Node and Electron experience. This was also the first time I've gone through the docs, and fresh eyes almost always catch some mistakes. However, if there are any changes you don't like, feel free to revert them or discuss them with me and I can explain my thought process behind them.

Also, I think we should remove the diagram on the introduction page as it is kinda ugly IMO, and it is all restated by the "How It Works" section below, and I think that does a much better job of explaining the workflow. I didn't remove it in this PR as I think some people may want to keep it so we should discuss that first.

### CODE REVIEW

File: website/docs/getting-started/intro.md [hunk 1]
```diff
@@ -5,88 +5,91 @@ title: 'Introduction'
...
+Here you will find instructions to setup and customize Tauri on your environment
 
-If you find something unclear, an error or simply would like to propose an improvement, you have several options:<br />
+If you find an error or something unclear, or would like to propose an improvement, you have several options:<br />
 
-1. Open an issue.<br />
+1. Open an issue on our [Github Repo](https://github.com/tauri-apps/tauri-docs).<br />
 2. Visit our [Discord server](https://discord.gg/SpmNs4S) and raise your concern.<br />
-3. Request to join one of the working groups to gain access (on Discord).
+3. Request to join the documentation working group on Discord to gain access to its discussion channel.
```
Comment (Laegel) [inline website/docs/getting-started/intro.md]:
```suggestion
3. Request to join the education working group on Discord to gain access to its discussion channel.
```

File: website/docs/getting-started/intro.md [hunk 2]
```diff
@@ -5,88 +5,91 @@ title: 'Introduction'
...
+Your app will need icons, so Tauri provides a command for making all the icons your app will need, no matter which platform you are targeting. Simply place a 1240x1240 png (with transparency) named `app-icon.png` in your project folder and run `tauri icon`. The icons will be placed in `src-tauri/icons` and automatically consumed by Tauri during the build process.
 
 ### Bundling an App
 
-The basic process of building a Tauri App is very straightforward and unopinionated. In the Tauri configuration file you reference a distribution folder e.g. `../dist/spa` where your HTML, CSS, JS and other assets can be found. Then you tell the Tauri CLI to build your app. Tauri prepares your code depending upon your configuration and bundles everything up into a nice and tidy binary.
+The basic process of building a Tauri App is very straightforward and unopinionated. In the Tauri configuration file you reference a distribution folder e.g. `../dist/spa` where your HTML, CSS, JS and other assets can be found. Then you run `tauri build`. Tauri prepares your code depending upon your configuration and bundles everything up into a nice and tidy binary.
```
Comment (Laegel) [inline website/docs/getting-started/intro.md]: People may not use `tauri build` to actually bundle their app but maybe `yarn tauri build`, `npm run tauri build` or another script. I think it would be better to keep this sentence in plain English.

File: website/docs/getting-started/intro.md [hunk 3]
```diff
@@ -5,88 +5,91 @@ title: 'Introduction'
...
+The first generation user interface in Tauri apps leverages Cocoa/WebKit on macOS, gtk-webkit2 on Linux and MSHTML (IE10/11) or Webkit via EdgeHTML / Chakra on Windows. **Tauri** leverages the MIT licensed prior work known as [webview](https://github.com/zserge/webview) that has been incorporated into [web-view](https://github.com/Boscop/web-view).
 
 <div class="alert alert--warning" role="alert">
   We are currently working on enabling the Web APIs and many of them may not be available on your platform.
 </div>
 
-## The Templates
+### The `src-tauri` Folder
+
+The `src-tauri` folder contains both the configuration for your Tauri app as well as any native Rust code. It is automatically created and filled with boilerplate code when `tauri init` is run. Configuration is stored in the `src-tauri/tauri.conf.json` file, and the `src-tauri/src` folder contains the native Rust code.
```
Comment (Laegel) [inline website/docs/getting-started/intro.md]:
```suggestion
The `src-tauri` folder contains both the configuration for your Tauri app as well as any native Rust code. It is automatically created and filled with boilerplate code when Tauri is initialized. Configuration is stored in the `src-tauri/tauri.conf.json` file, and the `src-tauri/src` folder contains the native Rust code.
```

File: website/docs/getting-started/intro.md [hunk 4]
```diff
@@ -5,88 +5,91 @@ title: 'Introduction'
...
+### Components of Tauri
 
-The node CLI has a bunch of "syntactic" sugar that sets up the dev env or prepares assets for the bundle process (as well as the tauri icon command for making icons.)
+The Node CLI reads your configuration file and gets everything prepped for bundling into a binary, such as configuring the injection of the Tauri API. It also contains a few helpful commands such `tauri icon` command for making icons, `tauri init` for scaffolding an app, and `tauri info` for debugging your environment.
 
-The next piece is the actual rust crate that binds web-view, constructs the webview and has the API (file read / write, etc.)
+The Tauri Rust Crate is the actual native code that binds to the Webview, creates your app's window, and provides the native API (file read / write, etc.).
 
-Finally, the bundler, which takes all the pieces and puts them together in a nice and easy to consume binary for the desktop OS's
+The bundler combines your HTML/JS/CSS, native rust code, and the Tauri Rust Crate into a nice and easy to consume binary for your target OS.
```
Comment (Laegel) [inline website/docs/getting-started/intro.md]:
```suggestion
The bundler combines your HTML/JS/CSS, native Rust code, and the Tauri Rust Crate into a nice and easy to consume binary for your target OS.
```

File: website/docs/usage/debugging.md [hunk 5]
```diff
@@ -38,33 +38,36 @@ For more information about this error, try `rustc --explain E0433`
...
-## Create a debugging BUILD
+## Create a Debug Build
 
-There are cases where you might need to inspect the JS console in the final bundle, so tauri provides a simple command to create a debugging bundle:
+There are cases where you might need to inspect the JS console in the final bundle, so Tauri provides a simple command to create a debugging bundle:
 
 ```
-local:  yarn tauri build --debug
-global: tauri build --debug
+local install:  yarn tauri build --debug
```
Comment (Laegel) [inline website/docs/usage/debugging.md:73]: Not sure why you added "install" here.

File: website/docs/usage/debugging.md [hunk 6]
```diff
@@ -38,33 +38,36 @@ For more information about this error, try `rustc --explain E0433`
...
+global install: tauri build --debug
```
Comment (Laegel) [inline website/docs/usage/debugging.md:74]: Same as above.

File: website/docs/usage/debugging.md (hunk 5 above)
Comment (Laegel) [reply website/docs/usage/debugging.md:73]: Okay, I get it.

File: website/docs/getting-started/intro.md (hunk 2 above)
Comment (nothingismagick) [reply website/docs/getting-started/intro.md]:
```suggestion
The basic process of building a Tauri App is very straightforward and unopinionated. In the Tauri configuration file you reference a distribution folder e.g. `../dist/spa` where your HTML, CSS, JS and other assets can be found. Then you tell the Tauri CLI to build your app. Tauri prepares your code depending upon your configuration and bundles everything up into a nice and tidy binary.
```

File: website/docs/usage/identity.md [hunk 7]
```diff
@@ -38,16 +38,17 @@ icon = [
...
 - icon.icns = MacOS
 - icon.ico = MS Windows
-- *.png = Linux
-
+- \*.png = Linux
 
 ## Splashscreen
 
-Bootstrapping an application may take time. Making users wait without explanations is bad for your UX; therefore, Tauri allows you to define a custom splashscreen, a placeholder that will be displayed in the first place until you think it can be swapped for the application content.
+Loading an application may take time. Making users wait without any indication of progress is bad for your UX. Tauri allows you to define a custom splashscreen, a placeholder that will be displayed until your application has finished loading.
```
Comment (Laegel) [inline website/docs/usage/identity.md]:
```suggestion
Loading an application for the first time may take time. Making users wait without any indication of progress is bad for your UX. Tauri allows you to define a custom splashscreen, a placeholder that will be displayed until your application has finished loading.
```

**Review by nothingismagick — COMMENTED** [review]
Looks great - let's just resolve these last couple comments and merge.

**Review by nothingismagick — APPROVED** [review]
